node/npm/glide/package.json

```diff
@@ -26,7 +26,7 @@
         "client",
         "valkey-glide"
     ],
-    "author": "Amazon Web Services",
+    "author": "Valkey contributors",
```

Comment:
Is this the right legal term?

Reply:
It could be "Valkey GLIDE Project Contributors" as we write in license headers

Reply:
Well, this one is not really catchy 😆
We need to check it.
All the subtle things need to be discussed, especially when it's legal things.

Reply:
Python client has "Valkey GLIDE Maintainers"

Reply:
Sounds better

Reply:
Updated everywhere